src/components/LicensePicker/utils.js

```diff
@@ -10,7 +10,12 @@ const NOASSERTION = 'NOASSERTION'
 // Shared methods appliable to License Picker
 export default class LicensePickerUtils {
   static parseLicense(license) {
-    return license && !['NONE', 'NOASSERTION'].includes(license) ? parse(license) : { license }
+    return license &&
+      !['NONE', 'NOASSERTION'].includes(license) &&
+      !license.includes('NOASSERTION') &&
+      !license.includes('NONE')
+      ? parse(license)
```

Comment:
Is it parse() that does not handle NOASSERTION?

We might want to sync our spdx parsing across service and website

Reply:
Yes, parse() fails when it contains a NOASSERTION value.
Has this been already solved on the service? I'm gonna check then.

Reply:
Are you referring to https://github.com/clearlydefined/service/blob/master/lib/spdx.js#L21 ?

Reply:
yep. 
I put up a repo at https://github.com/clearlydefined/spdx if we want to experiment with sharing the code from that spot

Reply:
What about the way to share the code? Should we use a simple `npm link` instruction or maybe publish a library on npm?

Reply:
we can publish to npm. `@clearlydefined/spdx`